src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -241,26 +284,30 @@ BinaryCompactObject::TimeDependence::default_value() noexcept {
 
 std::vector<std::array<size_t, 3>> BinaryCompactObject::initial_extents() const
     noexcept {
-  size_t number_of_blocks = 44;
-  if (not excise_interior_A_) {
-    number_of_blocks++;
-  }
-  if (not excise_interior_B_) {
-    number_of_blocks++;
-  }
-  return {number_of_blocks, make_array<3>(initial_grid_points_per_dim_)};
+  return {number_of_blocks_, make_array<3>(initial_grid_points_per_dim_)};
 }
 
 std::vector<std::array<size_t, 3>>
 BinaryCompactObject::initial_refinement_levels() const noexcept {
-  size_t number_of_blocks = 44;
-  if (not excise_interior_A_) {
-    number_of_blocks++;
-  }
-  if (not excise_interior_B_) {
-    number_of_blocks++;
+  std::vector<std::array<size_t, 3>> initial_levels{
+      number_of_blocks_, make_array<3>(initial_refinement_)};
+  // Increase the radial refinement of the blocks corresponding to the outer
+  // spherical shell (with sphericity == 1 throughout) to achieve the desired
+  // number of radial refinements. The outer layer consists of 10 blocks,
+  // created via wedge_coordinate_maps() with use_half_wedges == true. Because
+  // this outer layer of blocks is added last to the CoordinateMaps in
+  // create_domain(), the 10 blocks to refine are the last 10 blocks in the
+  // domain--unless one or both of the interiors are not excised. (For each
+  // interior not excised, there is one extra block appended to the list of
+  // CoordinateMaps.)
+  if (outer_layer_radial_refinements_) {
+    for (size_t block = 44; block < 54; ++block) {
+      // Refine in the radial direction, which is direction 2
+      // (i.e. the zeta direction)
+      gsl::at(initial_levels[block], 2) += outer_layer_radial_refinements_;
```

Comment:
What's not clear to me here is why this is a `+=` as opposed to an `=`. I would expect that the refinement is not the inner refinement+outer refinement, but just refinement. I'm also not clear if the wave zone starts at the cube-to-shell layer or the shell layer. I think it would be the shell layer only, because the other is a transition region.

Reply:
There’s a separate option to globally refine everywhere in all 3 directions. I want to additionally refine radially beyond that. Wouldn’t = instead of +=negate the InitialRefinementLevels global refinement in the radial direction?

Reply:
Yes it would. I'm fine with either leaving the option name as-is and changing `+=` to an `=`, or changing the name to `AdditionalOuter...` so it's clear what's happening.

Reply:
I’ll change the name. Good suggestion!